pkg/operator/operator.go

```diff
@@ -119,6 +133,21 @@ func (o *Operator) Start(stop <-chan struct{}) error {
 		return fmt.Errorf("failed to ensure default cluster ingress: %v", err)
 	}
 
+	// Start secondary caches.
+	for _, cache := range o.caches {
+		go func() {
+			if err := cache.Start(stop); err != nil {
+				// TODO: propagate to stop channel?
+				logrus.Infof("cache stopped with error: %v", err)
```

Comment:
Good point. Refactored to handle async cache failures. Wonder if we should crash if a cache returns at all regardless of error (since we don't restart the caches)...